README.md

```diff
@@ -36,19 +37,18 @@ Apache Geode includes the following features:
 * Cluster-wide configurations that can be persisted and exported to other clusters.
 * Remote cluster management through HTTP.
 * REST APIs for REST-enabled application development.
-* Rolling upgrade between major version releases.
+* Rolling upgrades may be possible, but they will be subject to any limitations imposed by new features.
```

Comment:
We have special handling of messages sent from members of older versions. I think it is safe to leave this as is.